src/Commands/HTTPS/SetCommand.php

```diff
@@ -15,22 +15,22 @@ class SetCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Add or replace an HTTPS certificate on an environment
+     * Enables HTTPS and/or updates the SSL certificate for the environment.
```

Comment:
The word 'certificate' was not capitalized here. It should all be the same way. (Lowercase, plz!)